ci-studio-common-util/cmd/root.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:          "ci-studio-common-util",
+	Short:        "Utility operations to manage the installation of ci-studio-common",
+	SilenceUsage: true,
+}
+
+// Execute does something?
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
```

Comment:
[nit] If you don't use `err`, you could do
```go
if rootCmd.Execute() != nil {
	os.Exit(-1)
}
```
That said, could this `err` be informative?